### ISSUE TITLE
NVDA 2016.4 and problem with menu bars

### ISSUE THREAD

**Issue by dawidpieper** [nvaccess/nvda#6650]
Sometimes, rarely, but, nevertheless, when we exit a window from the menu and the window will disappear when we're in menu bar, the NVDA hangs up. It reacts on commands sent by screenreaderapi and keycommands like NVDA+F12, however it doesn't read any opened windows. System works properly, restart of NVDA helps. I noticed this problem on Windows 7 Pro X64, SP1, in the window of Windows Mail and, then, after about 30 attempts, in Notepad. This Windows session have been running about 12 days, so it can be a problem. I suspect, this problem can be associated with menus that aren't focused properly by Windows after they appear.

**Comment by Brian1Gaff** [comment]
Are you sure this is a new issue? I have had the odd funny with non reading 
menu  pull downs in Goldwave for some versions now, and eventually put the 
issue down to goldwave not nvda as restarting the application cured it.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by ehollig** [comment]
@dawidpieper, can you reproduce this? If you can, it may be useful to see a log with debug logging enabled.

**Comment by Adriani90** [comment]
@dawidpieper are you still seeing this issue?

**Comment by ehollig** [comment]
Due to insufficient information to reproduce the issue, and several requests for more information from the original author without a response, I am going to close this issue. @dawidpieper if you can still reproduce this issue with NVDA 2019.2.1 and can provide a debug log with clear steps to reproduce the issue, we can reopen this ticket. Closing
